src/services/Api.ts

```diff
@@ -244,6 +244,10 @@ export const getIstioConfigValidations = (
   );
 };
 
+export const getWorkloads = (auth: string, namespace: String): Promise<Response<Validations>> => {
```

Comment:
I don't think this is correct, and I am afraid it can propagate some extra code.
Just following other API methods, the Response is forced to have the type:
```
i.e.
Response<ServiceList>
Response<IstioConfigList
```
I think for workloads this adds extra logic in the classes used to map the types.

So, I guess it should more or less follow the same pattern unless I'm missing something.